config/routes.rb

```diff
@@ -177,6 +177,11 @@
   resources :partnerships, only: %i[index create show], param: :option
   resources :display_ad_events, only: [:create]
   resource :pro_membership, path: :pro, only: %i[show create update]
+  resources :user_blocks do
```

Comment:
I suggest adding an `only` option because not all of the methods are used.

Reply:
Also, this way duplicate `show`, `create` and `destroy` routes are created. I suggest only keeping those we need.

Reply:
I was confused about the syntax 😅 Thanks for the heads up!